python/doc/source/parquet.rst

```diff
@@ -70,7 +70,9 @@ Let's look at a simple table:
                       'two': ['foo', 'bar', 'baz'],
                       'three': [True, False, True]},
                       index=list('abc'))
-   table = pa.Table.from_pandas(df)
+   table = pa.Table.from_pandas(df, preserve_index=False)
+
+We remove the dataframe index as we don't need it in the Parquet output file. If we left it in, we would end up with a column in our Parquet file called `__index_level_0__`, and it would inflate our output size.
```

Comment:
I agree with Uwe that we should leave this example as is and add a new example with a non-meaningful index (e.g. the default one). Rather than saying "as we don't need it" (which might confuse; why does the example set the index, then?) , instead say "_if_ the index is not used, it can be omitted from the file by..."